admin/class-wp-security-bp-admin.php

```diff
@@ -92,7 +92,7 @@ public function enqueue_styles( $hook_suffix ) {
 		 * class.
 		 */
 		if ( $hook_suffix === $this->hook_suffix ) {
-			wp_enqueue_style( $this->plugin_name, plugin_dir_url( __FILE__ ) . 'css/wp-security-bp-admin.css', array(), $this->version, 'all' );
+			wp_enqueue_style( $this->plugin_name, plugin_dir_url( __FILE__ ) . 'src/css/wp-security-bp-admin.css', array(), $this->version, 'all' );
```

Comment:
This should stay in `admin/css` folder, at this time we still don’t have any source files for css